### ISSUE TITLE
Years are returned as null

### ISSUE THREAD

**Issue by ZMYaro** [43081j/id3#55]
This library seems to read most tags fine, but always returns a `year: null` field, whether or not the song has a year in its tags.

**Comment by 43081j** [comment]
I've tried to fix it in #56 but MP3s vary so much i can't possibly know if it worked for you.

Could` you possibly clone the branch in that PR, `npm run build` and try it against your mp3 with something like this:

```ts
import * as id3 from './lib/id3.js';

id3.fromPath('./test.mp3').then((tags) => {
  console.log(tags);
});
```

I had it in a `test.mjs` which i then call directly with node `node test.mjs` (node v14).

**Comment by ZMYaro** [comment]
Sorry for taking so long to get back to the project I was using this on.  Yes, that fix worked for me; thank you so much!
